app/assets/stylesheets/vendor/main.css

```diff
@@ -310,12 +310,6 @@ h4 span { font-family: 'source_sans_pro_semiboldRg', arial, sans-serif; }
 
 a:hover { text-decoration: underline; color: #2b2b2b; }
 
-.message {
-    display: inline-block;
-    border: 1px solid #bebebe;
-    padding: .3em .5em;
-    margin: .3em 1em 1em 0 !important;
-}
```

Comment:
This class is not used for any other HTML elements, so its deletion should not cause formatting problems elsewhere on the site.